src/Views/Filter/Location.cshtml

```diff
@@ -83,14 +83,14 @@
 
                 <div class="govuk-radios__conditional govuk-radios__conditional--hidden @(Model.Errors.HasError(" query ") ? "form-group--error " : " ")" id="query-container" data-module="track-no-provider-results">
```

Comment:
Should this still be `query-container`? 🤔

Reply:
Ideally i'd like `query` to disappear completely because its too vague but I ended up down a rabbit hole doing that and just reverted to this simple, quicker change